packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js

```diff
@@ -95,6 +96,9 @@ class DurationEdit extends React.Component {
     this.props.onChange(calculateMilliseconds(hoursValue, minutesValue))
   }
 
+  handleOnBlur = () => this.setState({showUnits: (this.state.hours.length >= 1 || this.state.minutes.length >= 1)})
```

Comment:
When I execute the following steps only a `1` is shown:
1. Load the component.
2. Delete minutes. (Or hours respectively.)
3. Click outside.
I think there should be `1 hrs 0 min`.